pandas/io/gbq.py

```diff
@@ -440,8 +440,9 @@ def load_data(self, dataframe, dataset_id, table_id, chunksize):
             remaining_rows -= 1
 
             if (len(rows) % chunksize == 0) or (remaining_rows == 0):
-                self._print("\rStreaming Insert is {0}% Complete".format(
-                    ((total_rows - remaining_rows) * 100) / total_rows))
+                if self.verbose:
+                    self._print("\rStreaming Insert is {0}% Complete".format(
```

Comment:
`total_rows` is assigned after a check for `verbose` at line 430, so presently this fails because it can't evaluate the unassigned variable. If `self._print` is already checking for `verbose`, perhaps removing the conditional at line 429 and always assigning `total_rows` is the better move?